### PR TITLE
Cld 12475 refactor html for scroll into view spec

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: test/specs/ScrollingSpec.ts [hunk 1]
```diff
@@ -15,15 +15,17 @@ export namespace PageLocator {
 describeCommon('Scroll', () => {
   it('scrollIntoView ', () => {
     BrowserUtils.waitForDisplayed(PageLocator.HEADER);
-    assert.isFalse($(PageLocator.CAT_IMAGE).isDisplayedInViewport());
-    BrowserUtils.scrollIntoView(PageLocator.CAT_IMAGE);
-    assert.isTrue($(PageLocator.CAT_IMAGE).isDisplayedInViewport());
+    assert.isFalse($(PageLocator.LAST_TOP_LINK).isDisplayedInViewport());
```
Comment (FelixZilber) [inline test/specs/ScrollingSpec.ts]: Suggest to add test that scroll to an element that already in viewport (first element)

Comment (Ad1B3nAr0ya) [reply test/specs/ScrollingSpec.ts]: done

File: test/specs/ScrollingIntoViewSpec.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const HEADER: string = "//*[@id='top']/header/h1";
+const LAST_TOP_LINK: string =
+  "//*[@id='scrollingIntoView']//*[@id='scrollToElementId']";
+
+/**
+ * wdio-allure-ts Scrolling Into View test
+ */
+describeCommon('ScrollingIntoViewSpec', () => {
+  it('scrollIntoView to element outside of viewport', () => {
+    BrowserUtils.waitForDisplayed(HEADER);
```
Comment (FelixZilber) [inline test/specs/ScrollingIntoViewSpec.ts]: why do you need to wait for header?

File: test/specs/ScrollingIntoViewSpec.ts [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    assert.isFalse($(LAST_TOP_LINK).isDisplayedInViewport());
+    BrowserUtils.scrollIntoView(LAST_TOP_LINK);
+    assert.isTrue($(LAST_TOP_LINK).isDisplayedInViewport());
+  });
+
+  it('scrollIntoView to element in viewport', () => {
+    BrowserUtils.waitForDisplayed(HEADER);
```
Comment (FelixZilber) [inline test/specs/ScrollingIntoViewSpec.ts]: do you really need it ?

**Pull request merged**